services/workflows-service/src/project/project-scope.service.ts

```diff
@@ -73,59 +73,59 @@ export class ProjectScopeService {
     return args;
   }
 
-  scopeCreate<T>(args: Prisma.SelectSubset<T, PrismaGeneralInsertArgs>, projectIds?: TProjectIds) {
+  scopeCreate<T>(args: Prisma.SelectSubset<T, PrismaGeneralInsertArgs>, projectId?: TProjectId) {
     // @ts-expect-error - dynamically typed for all queries
     args.data = {
       // @ts-expect-error - dynamically typed for all queries
       ...args.data,
       project: {
         connect: {
-          id: projectIds?.[0],
+          id: projectId,
         },
       },
     };
 
     return args;
   }
 
-  scopeUpdate<T>(args: Prisma.SelectSubset<T, Prisma.FilterUpdateArgs>, projectIds?: TProjectIds) {
+  scopeUpdate<T>(args: Prisma.SelectSubset<T, Prisma.FilterUpdateArgs>, projectId: TProjectId) {
     args = this.scopeCreate(args);
     // @ts-expect-error - dynamically typed for all queries
     args.data = {
       // @ts-expect-error - dynamically typed for all queries
       ...args.data,
       project: {
         connect: {
-          id: projectIds?.[0],
+          id: projectId,
         },
       },
     };
 
     return args;
   }
 
-  scopeUpsert<T>(args: Prisma.SelectSubset<T, PrismaGeneralUpsertArgs>, projectIds?: TProjectIds) {
+  scopeUpsert<T>(args: Prisma.SelectSubset<T, PrismaGeneralUpsertArgs>, projectId?: TProjectId) {
```

Comment:
should receive both projectId ( not optional! ) and  projectIds -> 
as projectIds for where
and projectID for create/update